src/components/Models/components/LedLight.tsx

```diff
@@ -0,0 +1,104 @@
+import { useGLTF} from "@react-three/drei";
+import { useEffect, useRef } from "react";
+import { Vector3, InstancedMesh, Matrix4, Color, Euler } from "three";
+import { GLTF} from "three-stdlib";
+
+// types
```

Comment:
Add regions to make the code more readable

Reply:
I just checked the deployment branch too, here are some issues I found:

Must Fix -
The LED Lights are working as intended, but only the top row was made;
The Main Light is not turned on, it should be, for now at least, until we add the dark/light theme effects;

Future // If you want to -
The LED Lights, not the bloom, is strong enough to create the visual effects on the wall, but not on the table and the objects on it, we could create a proper shader, like cited on the issue #19 or just try to adjust the light color settings to achieve something similiar to the 3D render 
![image](https://github.com/XeretaDe/portfolio-cariello/assets/50390948/3d7d49e4-a97f-40fe-8843-53ec58a77cfb)